test/features/support/env.rb

```diff
@@ -28,3 +27,9 @@
 World(OllertTest::Navigation) do
   Ollert.new
 end
+
+Trello.configure do |config|
+  config.developer_public_key = ENV['INTEGRATION_KEY']
```

Comment:
Ok, and use `MEMBER_TOKEN` rather than `PRIVATE_TOKEN`? Also should the current `PUBLIC_KEY` be renamed as well and use `DEVELOPER_PUBLIC_KEY` for both?